### PR TITLE
Documentation for Work Management Process

### PR DESCRIPTION
**Pull request by sdhoyt**
# Change Summary

## Overview
This PR adds documentation for our new work management process and combines the SIT-4 planning effort with the Github release boards

### CODE REVIEW

File: docs/source/project-management/work-management.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Work Management Overview
+------------------------
+
+The SDC uses Github issues to track work. These issues are organized into two types of tasks: high-level tasks and implementation tasks.
+
+- High-level tasks
+    - used to track the progress of each instrument and data level
+    - tasks should be fairly static throughout development and will only change if an instrument changes their data products or levels
+    - used to track the progress of implementation tasks
+    - managed in the high-level board
```
Comment (bourque) [inline docs/source/project-management/work-management.rst]:
```suggestion
    - managed in the high-level board
    - labeled with the "Repo: Parent Issue" GitHub label
```

File: docs/source/project-management/work-management.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Backlog Grooming
+----------------
+
+The SDC holds a monthly backlog grooming prior to the start of each release. The purpose of the backlog grooming is to plan the work for the upcoming release and make our best attempt to create an executable plan for the release.
+The backlog grooming is also used to notify the team of any changes in tasks or work estimates that have not already been accounted for in the SIT planning.
+Any changes in time allocation should also be reported.
+These changes will be added to the SIT planning document to keep the projected timeline up to date.
+
+Before Backlog Grooming
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (bourque) [inline docs/source/project-management/work-management.rst]:
```suggestion
Before Backlog Grooming
~~~~~~~~~~~~~~~~~~~~~~~
```

**Review by bourque — APPROVED** [review]
Very nice! This looks good to me. This workflow is what I also envisioned with using these boards and labeling, but it would be good to hear what others think of the workflow overall.

**Comment by sdhoyt** [comment]
@bourque Thanks for the quick review!

**Comment by sdhoyt** [comment]
I don't think we talked about this previously, but I added to the documentation that the High-level tasks should also be used for infrastructure. I'm thinking one high-level task for each category of infrastructure work. So I'm thinking we'd have one Reprocessing high-level task and all the issues related to that would be sub-tasks.

**Review by tech3371 — APPROVED** [review]
Thank you for doing this!

File: docs/source/project-management/work-management.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Work Management Overview
+------------------------
+
+The SDC uses Github issues to track work. These issues are organized into two types of tasks: high-level tasks and implementation tasks.
+
+- High-level tasks
+    - used to track the progress of each instrument/data level and infrastructure features (ie. Reprocessing high-level task)
+    - tasks should be fairly static throughout development and will only change if an instrument changes their data products or levels
+    - used to track the progress of implementation tasks
+    - managed in the high-level board
```
Comment (maxinelasp) [inline docs/source/project-management/work-management.rst]: Could you name the high-level board here? We have a lot of boards and I forget which ones are which

File: docs/source/project-management/work-management.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    - labeled with the "Repo: Parent Issue" GitHub label
+- Implementation tasks
+    - define work that needs to be implemented
+    - each task should be small enough to be handled in a single PR
+    - organized as sub-tasks under high-level tasks
+    - should be added to high-level tasks as soon as they are identified
+    - should have work estimates tied to each task
```
Comment (maxinelasp) [inline docs/source/project-management/work-management.rst]:
```suggestion
    - should have work estimates tied to each task in real-time estimates
```

File: docs/source/project-management/work-management.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    - managed in the release boards
```
Comment (maxinelasp) [inline docs/source/project-management/work-management.rst:25]: Do we need each task to be part of a release board when it's made, or is it like, per month you add them?

File: docs/source/project-management/work-management.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+High-level Board
+~~~~~~~~~~~~~~~~
+
+The `high-level board <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/32>`_ contains a task for each instrument and data level. The tasks showing in this board should be fairly static throughout development and would only change if a instrument changes their data products or levels. This board provides a quick view of the progress being made across instruments.
+
+Release Board
+~~~~~~~~~~~~~
+
+The `release board <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/30>`_ allows for more detailed planning for the the current and upcoming release.
+Any tasks in the current release board that are not completed by the end of the release will be pushed to the next release.
```
Comment (maxinelasp) [inline docs/source/project-management/work-management.rst]:
```suggestion
Any tasks in the current release board that are not completed by the end of the release will be pushed to the next release, but it acts as an estimate for work that will be completed that month.
```

**Review by maxinelasp — APPROVED** [review]
Looks good! I made a few suggestions/comments.

File: docs/source/project-management/work-management.rst (hunk 5 above)
Comment (sdhoyt) [reply docs/source/project-management/work-management.rst:25]: No, there will be a 2 month rolling window for the release boards, so only tasks planned for those releases will be on boards. If you have an issue for GLOWS L2 that you're not ready to work on in the Oct or Nov, then that issue will only be tracked by you under your GLOWS L2 high-level task, not on any boards. The goal is to keep track of issues (under the high-level tasks) without having to plan when you'll work on them too far ahead of time with the boards.

I will try to make that more clear in the documentation.

File: docs/source/project-management/work-management.rst (hunk 6 above)
Comment (sdhoyt) [reply docs/source/project-management/work-management.rst]: >but it acts as an estimate for work that will be completed that month.

Can you clarify what you mean by that? I'm not sure I'm understanding the intent

Comment (maxinelasp) [reply docs/source/project-management/work-management.rst]: Yeah, I'm thinking I just want to make it clear that the monthly release board is a work estimate for that month, and doesn't just include everything

File: docs/source/project-management/work-management.rst (hunk 5 above)
Comment (maxinelasp) [reply docs/source/project-management/work-management.rst:25]: Sounds good, yeah I'd add a bullet point that they are not necessarily part of a release

File: docs/source/project-management/work-management.rst (hunk 6 above)
Comment (sdhoyt) [reply docs/source/project-management/work-management.rst]: Ah gotcha sounds good. Thanks for the review!

**Pull request merged**